artifacts/checksums.txt

```diff
@@ -1,3 +1,18 @@
-c3d45ee3e0dc6e11e35d8b6ae0462d486ebd23b14203e5bc6af4d79b06497838  astroport_airdrop.wasm
-81b21c3ab14f052b0b014cc33cadf7a31903204f3a79fc5559b056d4b6f815bd  astroport_auction.wasm
-1f6e5b72504ae2bd06c5e67a61c39e1e3a16230e61f699a272348503f5467be2  astroport_lockdrop.wasm
+68a0e4e4874b41f28b873c8298f19f2209ca7a61c061e11694ea02ed6a4a602d  astro_airdrop.wasm
+cfc0dc1ff384de92e6e931c61552c8b26dc492ed6b15dfc4cdb22be125a4c25a  astro_auction.wasm
+edbae148d21dbd102d0a039ad9459dccb32f2334861bac74fd150825eb264113  astro_bootstrap_auction.wasm
```

Comment:
I think we should have here only astroport-{airdrop,auction,lockdrop} checksums of their wasm files. Other astroport contracts should be built at their repository. Terraswap contracts with their test cases are built here: https://github.com/astroport-fi/terraswap. Because astroport repository is now public I suggest to use basic build script in build_release.sh:
`docker run --rm -v "$(pwd)":/code \
  --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
  cosmwasm/workspace-optimizer:0.12.4`

Reply:
yes makes sense. corrected